backend/analysis_framework/tests/test_apis.py

```diff
@@ -0,0 +1,207 @@
+from rest_framework.test import APITestCase
+from user.tests.test_apis import AuthMixin
+from rest_framework import status
+from analysis_framework.models import (
+    AnalysisFramework, Widget,
+    Filter, Exportable
+)
+
```

Comment:
Let's write a "TODO: test_permissions" (or actually write them). We have been using `test_permissions.py` file to test for permissions.